code/modules/cargo/exports.dm

```diff
@@ -99,12 +116,14 @@ Credit dupes that require a lot of manual work shouldn't be removed, unless they
 // Adds item's cost and amount to the current export cycle.
 // get_cost, get_amount and applies_to do not neccesary mean a successful sale.
 /datum/export/proc/sell_object(obj/O, contr = 0, emag = 0)
-	var/cost = get_cost(O)
+	var/the_cost = get_cost(O)
 	var/amount = get_amount(O)
-	total_cost += cost
+	total_cost += the_cost
 	total_amount += amount
+	
+	cost -= (the_cost * (0.01*amount))
```

Comment:
Why are you multiplying the_cost by amount? get_cost already multiplies by amount, so this line is effectively subtracting cost * amount^2  * 0.01

Hitting the bottom cost limit is incredibly fast. In fact cost will reach 1 at amount=10